src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitOperationDao;
+import core.basesyntax.dao.impl.FruitOperationDaoImpl;
+import core.basesyntax.io.FileReader;
+import core.basesyntax.io.FileWriter;
+import core.basesyntax.model.FruitReport;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitReportService;
+import core.basesyntax.service.ParseRowToObjectService;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.service.impl.FruitReportServiceImpl;
+import core.basesyntax.service.impl.ParseRowToObjectServiceImpl;
+import core.basesyntax.service.impl.TransactionServiceImpl;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        Map<String, BigDecimal> inventory = new HashMap<>();
+        FileWriter writer = new FileWriter();
+        FileReader reader = new FileReader();
+        FruitOperationDao fruitOperationDao = new FruitOperationDaoImpl(reader, writer);
```

Comment:
see what DAO is
https://www.baeldung.com/java-dao-pattern
dao does not need FileWriter or FileReader. DAO is for data base access